### PR TITLE
Add flag for startup completed

### PR DESCRIPTION
**Pull request by crobibero**

### CODE REVIEW

**Review by cvium — COMMENTED** [review]
How useful is this? Until CoreStartupHasCompleted == true, the entire API returns a 503.

**Comment by dkanada** [comment]
Oh, is this not the startup wizard? I meant that a flag for the startup wizard would be useful.

File: MediaBrowser.Model/System/PublicSystemInfo.cs [hunk 1]
```diff
@@ -39,5 +39,11 @@ public class PublicSystemInfo
         /// </summary>
         /// <value>The id.</value>
         public string Id { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether startup is completed.
+        /// </summary>
+        /// <value>The startup completion status.</value>
+        public bool StartupCompleted { get; set; }
```
Comment (cvium) [inline MediaBrowser.Model/System/PublicSystemInfo.cs:47]: Rename to StartupWizardCompleted?

**Review by cvium — APPROVED** [review]

**Comment by Bond-009** [comment]
What's the use case for this?

**Comment by crobibero** [comment]
> What's the use case for this?

Web needs this so it can show the startup wizard instead of the login screen if needed. Previously, server would redirect web to the startup wizard

**Review by Bond-009 — APPROVED** [review]

**Pull request merged**